packages/golden_toolkit/example/test/flutter_demo_page_test.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:golden_toolkit/golden_toolkit.dart';
+
+// ignore: avoid_relative_lib_imports
```

Comment:
Rather than ignore this lint I thin you can just import the package.

Reply:
`flutter_demo_page.dart` is exported by the `example.dart` barrel file. So i will change this import to be `import 'package:example/example.dart';` and remove the ignore

Reply:
So the analysis options succeeds locally, but your pipeline just failed with:
`error • Target of URI doesn't exist: 'package:example/example.dart' •`

So i think i'm going to have to shift back to relative imports to fix this. @fatherOfLegends